app/bundles/CoreBundle/Controller/CommonController.php

```diff
@@ -544,10 +544,14 @@ protected function setListFilters($name = null)
         }
         $name = 'mautic.'.$name;
 
+        if (false === $this->request->query->has('orderby') && false === $session->has("$name.orderbydir")) {
+            $session->set("$name.orderbydir", method_exists($this, 'getDefaultOrderDirection') ? $this->getDefaultOrderDirection() : 'ASC');
```

Comment:
Can we instead of `method_exists` check define the `getDefaultOrderDirection` method in the `CommonController` with default value of `ASC` and overwrite the method in the controllers who inherit from the `CommonController` and needs `DESC`? It would be a bit cleaner solution.